transmitter.js

```diff
@@ -1,16 +1,20 @@
-let transmissionFailureCount = 0;
+
+let transmissionFailureCount=0
+//let count = 0;
 
 function networkTransmitStub(celcius) {
     console.log(`Temperature to transmit: ${celcius} celcius`);
-
+    randomNumber=Math.random();
     // The intention here is to transmit the temperature over a network.
     // However, this is a stub, so we just print the message above
     // and give back a return code.
 
     // Return 200 for ok
     // Return 500 for not-ok
     // This stub always succeeds and returns 200
-    return 200;
+   
+    // random return
+    return randomNumber<0.5?500:200;

```

Comment:
This may sometimes end up giving random numbers, all > 0.5?
It's a good idea to randomize test data (which is a technique used in 'property-based testing'), but try to bring in some determinism here. How to ensure at least one error, to satisfy your expect?

Reply:
First time while returning I will return one error after that random returns I will generate

Reply:
Actually, the stub is part of the test code, it should not be mixed with `transmitInCelcius` (which is production code)
But if you put the stub in the test file, then you need to require the test file here. But the test file requires this file already, in order to test it.
Hint to resolve this: lookup "dependency injection"